extensions/wikia/FeedsAndPosts/FeedsAndPosts.setup.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Feeds and Posts
+ * Injects discussion feed into an article page
+ */
+
+// Autoload
+$wgAutoloadClasses['FeedsAndPostsHooks'] =  __DIR__ . '/FeedsAndPostsHooks.class.php';
+
+// Hooks
+$wgHooks['OasisSkinAssetGroups'][] = 'FeedsAndPostsHooks::onOasisSkinAssetGroups';
+
+$wgResourceModules['ext.wikia.feedsAndPosts'] = [
+	'scripts' => 'js/feedsAndPosts.js',
```

Comment:
If you're using an RL module you'll need also `'localBasePath' => __DIR__` and `'remoteExtPath' => 'wikia/FeedsAndPosts'` configuration options in the declaration.